assets/templates/accounts/associations_base.html

```diff
@@ -1,30 +1,30 @@
 {% extends 'base.html' %}
 
 {% block content %}
-
-        <h2 class="row">{{ association }}</h2>
-
-        <ul class="row nav nav-tabs">
-            <li class="nav-item">
-
-                <a class="nav-link{% block tab_overview %}{% endblock %}"
-                   href="{% url 'association_overview' association_name=association.slug %}">Overview
-                    {% if notify_overview %}<span class="badge badge-pill badge-warning">!</span>{% endif %}
-                </a>
-
-            </li>
-            <li class="nav-item">
-                <a class="nav-link{% block tab_credits %}{% endblock %}"
-                   href="{% url 'association_credits' association_name=association.slug %}">Credits</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link{% block tab_members %}{% endblock %}"
-                   href="{% url 'association_members' association_name=association.slug %}">Members</a>
-            </li>
-        </ul>
+        <div class="row">
+            <h2 class="col-12 pl-0">{{ association }}</h2>
```

Comment:
Like you mentioned yourself, content should be inside of columns for grid layouts so the correct way is to put the `h2` inside a `div` with the col class, then you won't need the `pl-0`.

But like for almost all other uses of the grid layout in this PR it is unnecessary and you can just put down `<h2>{{association}}</h2>` instead of the three lines.